docs/core/install/linux-ubuntu.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Install .NET Core on Ubuntu - .NET Core
+description: Demonstrates the various ways to install .NET Core SDK and .NET Core Runtime on Ubuntu.
+author: thraka
+ms.author: adegeo
+ms.date: 06/01/2020
+---
+
+# Install .NET Core SDK and .NET Core Runtime on Ubuntu
+
+.NET Core is supported on Ubuntu. This article describes how to install .NET Core on Ubuntu. When an Ubuntu version falls out of support, .NET Core is no longer supported with that version. However, these instructions may help you to get .NET Core running on those versions, even though it isn't supported.
+
+## Supported distributions
+
+The following is a list of currently supported .NET Core releases and the versions of Ubuntu they're supported on. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Ubuntu reaches end-of-life](https://wiki.ubuntu.com/Releases).
+
+For the best compatibility, choose a long-term release (LTS) version of both Ubuntu and .NET Core.
+
+| .NET Core   | Ubuntu version                                           |
+|-------------|----------------------------------------------------------|
+| 2.1 (LTS)   | [16.04 (LTS) ✔️](#1604-), [18.04 (LTS) ✔️](#1804-), [19.10 ❌](#1910-), [20.04 (LTS) ✔️](#2004-) |
+| 3.1 (LTS)   | [16.04 (LTS) ✔️](#1604-), [18.04 (LTS) ✔️](#1804-), [19.10 ❌](#1910-), [20.04 (LTS) ✔️](#2004-) |
+| 5.0 Preview | [16.04 (LTS) ✔️](#1604-), [18.04 (LTS) ✔️](#1804-), [19.10 ❌](#1910-), [20.04 (LTS) ✔️](#2004-) |
```

Comment:
Why ❌? I think it *might* be ok to omit a ✔️ to potentially drive usage towards LTS Ubuntus, but putting it on the same terms as lack of support doesn't seem right. Imagine an ❌ next to 20.10 as the rightmost entry for example.

Reply:
Whoops, yes, ❌ shouldn't be there.